commands/channel.go

```diff
@@ -345,32 +350,43 @@ func makeChannelPrivateCmdF(c client.Client, cmd *cobra.Command, args []string)
 }
 
 func renameChannelCmdF(c client.Client, cmd *cobra.Command, args []string) error {
-	var newDisplayName, newChannelName string
+	existingTeamChannel := args[0]
 
-	if len(args) < 2 {
-		return errors.New("Not enough arguments.")
+	newChannelName, err := cmd.Flags().GetString("name")
+	if err != nil {
+		return errors.New("Cannot get name flag, error : " + err.Error())
```

Comment:
Let's just directly return the error. It will be of the form: `fmt.Errorf("flag accessed but not defined: %s", name)`. So the name is already there.